quantum-adapters/kiwoom-adapter/src/kiwoom_api/api/news.py

```
@@ -0,0 +1,208 @@
+"""뉴스 조회 API 엔드포인트
+
+종목별 뉴스 조회, 카테고리별 뉴스 필터링, 뉴스 상세 정보 제공
+"""
+
+import sys
+from pathlib import Path
+from fastapi import APIRouter, HTTPException, Query, Path
```

Ruff (0.12.2)

Redefinition of unused `Path` from line 7

(F811)